tests/S3/Crypto/HeadersMetadataStrategyTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
```

Comment:
The test classes should probably be in a namespace corresponding to the directory structure.